frontend/src/metabase/dashboard/selectors/selectors.js

```diff
@@ -120,7 +120,7 @@ export const getDashboardComplete = createSelector(
       return null;
     }
 
-    const ordered_cards = dashboard.ordered_cards
+    const orderedDashcards = dashboard.dashcards
```

Comment:
I think this should just be `dashcards`.

Reply:
We need to distinguish it from `dashcards`, which is the name of the argument.

Reply:
It could be `newDashcards` too, but `orderedDashcards` feels more fitting because the most significant part of this transformation is making the dashcards are ordered.

Reply:
Works for me.